endpoints/cookie_sync.go

```diff
@@ -246,6 +242,19 @@ func (req *cookieSyncRequest) filterForPrivacy(permissions gdpr.Permissions, pri
 	}
 }
 
+func (req *cookieSyncRequest) filterForCCPA(privacyPolicies privacy.Policies, enforceCCPA bool) {
+	if !enforceCCPA {
+		return
+	}
+
+	for i := 0; i < len(req.Bidders); i++ {
+		if privacyPolicies.CCPA.ShouldEnforce(req.Bidders[i]) {
```

Comment:
It's not currently possible to specify a nosale bidders array via the cookie sync endpoint, but I'm wiring this up properly in case that changes in the future.